src/Reflection/Argument/Quantity.agda

```diff
@@ -0,0 +1,28 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Argument quantities used in the reflection machinery
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Reflection.Argument.Quantity where
+
+open import Relation.Nullary
+open import Relation.Binary
+open import Relation.Binary.PropositionalEquality
+
+------------------------------------------------------------------------
+-- Re-exporting the builtins publically
```

Comment:
Spelling "publicly"

Reply:
You may want to edit `Reflection.Argument.Information`: I copied this text from that file.